### DISCUSSION TITLE
Digital Ocean deployment not creating POSTGRES tables

### DISCUSSION THREAD

**Discussion by ghost** [keystonejs/keystone#7393] [Questions] [answered]
This seems to be a persistent issue but I can't find a solution, and it seems to fall into the bug category as I've used the example app from start to finish to test deploying to DO.

Using `yarn create keystone-app` which installs `1.0.1`.

Putting it onto digital ocean, and connecting to a postgresql 14 db - connecting works, but no tables are created.

I've tried a lot of different things, including the advised command of 

```
yarn
yarn keystone build
yarn keystone prisma migrate deploy
```

**Comment by raveling** [comment]
@kennedybaird are you running `prisma migrate deploy` against the Database on Digital Ocean?

**Comment by Achisingh** [comment, marked as answer]
Hi @kennedybaird,

Could you make sure you have done the following:

1. Set `useMigrations: true`. Refer to https://keystonejs.com/docs/guides/cli#about-database-migrations
2. Run `keystone dev` locally which will prompt you to create a migration.
3. Then you can run `keystone prisma migrate deploy` against your production database.
 
If that doesn't work, please show the output of these commands.

**Reply by ghost** [reply]
Still struggling with this.

This is the error I receive when I run `keystone prisma migrate deploy`

```
Error: P3019

The datasource provider `postgresql` specified in your schema does not match the one specified in the migration_lock.toml, `sqlite`. Please remove your current migration directory and start a new migration history with prisma migrate dev. Read more: https://pris.ly/d/migrate-provider-switch
```

**Reply by emmatown** [reply]
I would guess that you're trying to use SQLite in development and Postgres in production? If so, that isn't supported, you should use the same database provider in development and production
